### ISSUE TITLE
opentracing reports 500 when a route was not found

### ISSUE THREAD

**Issue by aryszka** [zalando/skipper#1145]
**Describe the bug**
When the route lookup doesn't return a route, skipper returns 404 by default, or the default status code set from the startup options, but 500 is sent to opentracing instead.

**To Reproduce**
Add context, for example:
1. start skipper with no routing configured, but with opentracing enabled
2. make any http request to skipper
3. skipper should respond with 404
4. check the matching trace in the used opentracing provider

**Expected behavior**
the trace has recorded 404 as the http status tag

**Observed behavior**
the trace has recorded 500

**Comment by szuecs** [comment]
skipper without route
```
% ./bin/skipper
[APP]INFO[0000] Expose metrics in codahale format
[APP]WARN[0000] no route source specified
[APP]WARN[0000] NewLocalRatelimit is deprecated, please use NewClientRatelimit
[APP]INFO[0000] Run secrets path refresher every 10m0s
[APP]INFO[0000] support listener on :9911
[APP]INFO[0000] proxy listener on :9090
[APP]INFO[0000] TLS settings not found, defaulting to HTTP
::1 - - [31/Jul/2019:13:51:30 +0200] "GET / HTTP/1.1" 404 10 "-" "curl/7.49.0" 0 localhost:9090 - -
::1 - - [31/Jul/2019:13:51:32 +0200] "GET / HTTP/1.1" 404 10 "-" "curl/7.49.0" 0 localhost:9090 - -
```

client:
```
% curl localhost:9090 -v
* Rebuilt URL to: localhost:9090/
*   Trying ::1...
* Connected to localhost (::1) port 9090 (#0)
> GET / HTTP/1.1
> Host: localhost:9090
> User-Agent: curl/7.49.0
> Accept: */*
>
< HTTP/1.1 404 Not Found
< Content-Type: text/plain; charset=utf-8
< Server: Skipper
< X-Content-Type-Options: nosniff
< Date: Wed, 31 Jul 2019 11:51:32 GMT
< Content-Length: 10
<
Not Found
* Connection #0 to host localhost left intact
```

**Comment by rbarilani** [comment]
@aryszka I can't reproduce the behavior, I thought that I have fixed it "accidentally" with this one #1719 but actually, the span was already tagged with the right `http.status_code`. Any thoughts? Might this issue be already fixed?
